apps/core/scenes/search/Datepickers.js

```diff
@@ -69,27 +88,35 @@ class Datepickers extends React.Component<Props, State> {
   };
 
   render() {
-    const { tripType, dateFrom, returnDateFrom } = this.props;
+    const { tripType, dateFrom, returnDateFrom, layout } = this.props;
+    const rowLayout = layout >= LAYOUT.largeMobile;
+    const returnType = tripType === 'return';
     const datePickerDate =
       this.state.selectDate === DATEPICKER_MODE.DEPARTURE
         ? dateFrom
         : returnDateFrom;
+
     return (
       <>
-        <TripInput
-          onPress={this.handleDepartureDatePress}
-          label="Departure"
-          icon={<Icon name="calendar" />}
-          value={format(dateFrom, DATE_FORMAT)}
-        />
-        {tripType === 'return' && (
+        <PickersWrapper layout={layout}>
           <TripInput
-            onPress={this.handleReturnDatePress}
-            label="Return"
+            style={rowLayout && styles.rowInput}
+            onPress={this.handleDepartureDatePress}
+            label="Departure"
             icon={<Icon name="calendar" />}
-            value={format(returnDateFrom, DATE_FORMAT)}
+            value={format(dateFrom, DATE_FORMAT)}
           />
-        )}
+          {(returnType || Platform.OS === 'web') && (
```

Comment:
Maybe you could do this check before the return, it would be easier to read. 

`const showReturnInput = tripType === 'return' || Platform.OS === 'web'`

Reply:
Updated. It's definitely more readable. Thanks